Sources/XcodesKit/XcodeSelect.swift

```diff
@@ -153,13 +160,16 @@ public func selectXcodeAtPath(_ pathString: String) -> Promise<ProcessOutput> {
 public enum XcodeSelectError: LocalizedError {
     case invalidPath(String)
     case invalidIndex(min: Int, max: Int, given: String?)
+    case noInstalledXcodes
 
     public var errorDescription: String? {
         switch self {
         case .invalidPath(let pathString):
-            return "Not a valid Xcode path: \(pathString)"
+            "Not a valid Xcode path: \(pathString)"
         case .invalidIndex(let min, let max, let given):
-            return "Not a valid number. Expecting a whole number between \(min)-\(max), but given \(given ?? "nothing")."
+            "Not a valid number. Expecting a whole number between \(min)-\(max), but given \(given ?? "nothing")."
+        case .noInstalledXcodes:
+            "No Xcode version installed. Please run 'xcodes install' and try again."
```

Comment:
I love the new switch as expression syntax as much as the next person, but is Xcodes using Swift 5.9 so we can take advantage of this?

Reply:
@MattKiazyk 👀